incubator/hnc/internal/reconcilers/object_test.go

```diff
@@ -231,6 +231,118 @@ var _ = Describe("Exceptions", func() {
 			})
 		}
 	})
+
+	Context("Update the descendant namespaces after 'select' exception annotation is set", func() {
```

Comment:
Add the issue number in the context (I've just submitted a change to make it easy to run the tests for a single issue)

Reply:
I added the issue number to the `It` block since 2 out 5 test cases are for issue #1448 and the `HNC_FOCUS` is on e2e tests (integration and unit tests are fast so no focus test is needed).

Reply:
Oh right :)